### PR TITLE
ICE dump: Blocked CI edition

### PR DESCRIPTION
**Pull request by fanninpm**
The fact that the CI fails as of now (2020-12-28 22:14 EST) should hopefully be unrelated to these ICEs.

### CODE REVIEW

**Comment by camelid** [comment]
CI is probably failing because of all of the ICEs that were fixed by `min_const_generics` but haven't been updated in glacier yet.

**Comment by fanninpm** [comment]
@JohnTitor and/or @Alexendoo Let me know if/when I need to push a blank commit.

File: ices/80371.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```
Comment (JohnTitor) [inline ices/80371.sh:1]: We don't have to use these flags and sh file at all. Just enough:
```rust
pub struct Header<'a> {
    pub value: &'a [u8],
}

pub fn test() {
    let _headers = [Header{value: &[]}; 128];
}

fn main() {}

```

File: ices/80409.rs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+use std::marker::PhantomData;
```
Comment (JohnTitor) [inline ices/80409.rs:1]: There are some hard tabs, please run rustfmt for this.

**Review by JohnTitor — CHANGES_REQUESTED** [review]
Thanks!
Left some reviews and merged the master branch to pass CI.

File: ices/80409.rs (hunk 2 above)
Comment (fanninpm) [reply ices/80409.rs:1]: Fixed in 126fd7f

**Review by JohnTitor — APPROVED** [review]

**Pull request merged**